src/widgets/ProfileWidget/ProfileWidget.scss

```diff
@@ -58,4 +58,55 @@
             color: #fff;
         }
     }
+
+    &__content {
+        padding: 0 var(--page-padding);
+    }
+
+    &__search-bar-container {
+        position: relative;
+        padding: 1.5rem 0rem;
+    }
+
+    &__search-bar-icon {
+        position: absolute;
+        top: 2.2rem;
+        left: 1rem;
+        width: 1.125rem;
+        height: 1.125rem;
+
+        & svg {
```

Comment:
https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md#please-do-not

Reply:
:heavy_check_mark: